docs/agents/acti_camera.rst

```diff
@@ -0,0 +1,83 @@
+.. highlight:: rst
+
+.. _acti_camera:
+
+====================
+ACTi Camera Agent
+====================
+
+The ACTi Camera Agent is an OCS Agent which grabs screenshots from ACTi cameras
+and saves files to a directory.
+
+.. argparse::
+    :filename: ../socs/agents/acti_camera/agent.py
+    :func: add_agent_args
+    :prog: python3 agent.py
+
+Configuration File Examples
+---------------------------
+
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container.
+
+OCS Site Config
+```````````````
+
+To configure the ACTi Camera Agent we need to add a ACTiCameraAgent
+block to our ocs configuration file. Here is an example configuration block
+using all of the available arguments::
+
+      {'agent-class': 'ACTiCameraAgent',
+       'instance-id': 'cameras',
+       'arguments': [['--mode', 'acq'],
+                     ['--camera-addresses', ['10.10.10.41', '10.10.10.42', '10.10.10.43']],
+                     ['--locations', ['location1', 'location2', 'location3']],
+                     ['--user', 'admin'],
+                     ['--password', 'password']]},
+
+.. note::
+    The ``--camera-addresses`` argument should be a list of the IP addresses
+    of the cameras on the network.
+    The ``--locations`` argument should be a list of names for camera locations.
+    This should be in the same order as the list of IP addresses.
+
+Docker Compose
+``````````````
+
+The iBootbar Agent should be configured to run in a Docker container. An
+example docker-compose service configuration is shown here::
+
+  ocs-acti-camera:
```

Comment:
Convention for service name is `ocs-<instance-id>`, this is for the Host Manager to parse things correctly.